docs/templates/properties.js

```diff
@@ -17,6 +17,22 @@ module.exports.anchor = function anchor({ item, contract }) {
   return res;
 };
 
+module.exports.fullname = function fullname({ item }) {
```

Comment:
Would this be a better name? Not sure if I'm getting it wrong

```suggestion
module.exports.fullanchor = function fullanchor({ item }) {
```